Doc/library/os.rst

```diff
@@ -3656,9 +3656,10 @@ written in Python, such as a mail server's external command delivery program.
 
 .. data:: EX_OK
 
-   Exit code that means no error occurred.
+   Exit code that means no error occurred. May be taken from the defined value of
+   `EXIT_SUCCESS` on some platforms. Generally has a value of zero.
```

Comment:
```suggestion
   ``EXIT_SUCCESS`` on some platforms. Generally has a value of zero.
```

Sorry, my bad in the suggestion. This is ReST, so it needs double backticks for an unspecified item.

Reply:
Oh! - I usually use triple, or single in markdown. - Now double? :O